src/Compilers/Core/MSBuildTask/Microsoft.CSharp.Core.targets

```diff
@@ -97,6 +97,7 @@
          Features="$(Features)"
          FileAlignment="$(FileAlignment)"
          GeneratedFilesOutputPath="$(CompilerGeneratedFilesOutputPath)"
+         GeneratedArtifactsOutputPath="$(CompilerGeneratedArtifactsOutputPath)"
```

Comment:
This likely needs defaults in `Microsoft.Managed.Core.targets`. See what that file does for `CompilerGeneratedFilesOutputPath`.

Reply:
i honestly don't undrestand what's going on in `Microsoft.Managed.Core.targets`.  Also, i don't really see these options as being similar here.  One option (CompilerGeneratedFilesOutputPath) is about dumping an internal detail of the compiler to disk or not to help assist the user.  The other option is just about if artifact generation is active or not.  It's unclear to me why there needs to be any special logic there.  Either you are using artifact generation or not.

Reply:
Hmm. I suppose if there's no global switch here like there is for generated files nothing needs to be done.